.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+on: 
+  push:
+  pull_request:
+    branches:
+      - development
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+      - run: npm ci
+      - run: npm run front-end:test
+      - run: npm run back-end:test
+  lint: 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+      - run: npm ci
+      - run: npm run front-end:lint
+      - run: npm run back-end:lint
+  scan:
+      name: npm audit
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+        - name: install dependencies
+          run: npm ci
+        - uses: oke-py/npm-audit-action@v1.8.2
+          with:
+            audit_level: moderate
+            github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't have access to this project's settings to set up the secrets, but since this ran, I think someone had already set up `GITHUB_TOKEN`

Reply:
if we're not wanting to open any issues or comments, then I wonder if we need `oke-py/npm-audit-action` or  github_tokens? Wonder if we can get what we need with `$ npm audit --audit-level=moderate` or something to that effect?